samples/Azure.Management.Storage/Generated/RestOperations/BlobContainersRestOperations.cs

```diff
@@ -693,13 +693,10 @@ internal HttpMessage CreateCreateOrUpdateImmutabilityPolicyRequest(string subscr
                 request.Headers.Add("If-Match", ifMatch);
             }
             request.Headers.Add("Accept", "application/json");
-            if (data != null)
-            {
-                request.Headers.Add("Content-Type", "application/json");
-                var content = new Utf8JsonRequestContent();
-                content.JsonWriter.WriteObjectValue(data);
-                request.Content = content;
-            }
+            request.Headers.Add("Content-Type", "application/json");
```

Comment:
throw exception if data is null?

Reply:
We assert the data is not null in the corresponding client method. Therefore we no longer need to assert again